src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
-body {
+* {
   margin: 0;
```

Comment:
It's better to use for `*` selector `box-sizing` rule only. `margin` you can set for any other selectors you need